refactor: clean up explicit standalone flags from tests

crisbeto opened the pull request:
Since standalone is the default, we can dropn the `standalone: true` flags from our tests.

JeanMeche reviewed (approved):
You beat me to it, I wanted to do that too. (An maybe also provide a migration that would finally remove all the `standalone: true` props in projects.

JeanMeche commented on packages/compiler-cli/test/ngtsc/debug_transform_spec.ts:
```diff
@@ -1200,14 +1198,12 @@ runInEachFileSystem(() => {
                 selector: 'child-component',
-                standalone: true,
-                template: ''
+                                template: ''
```
nit: The indent got messed up here

crisbeto committed:
refactor: clean up explicit standalone flags from tests

Since standalone is the default, we can dropn the `standalone: true` flags from our tests.

thePunderWoman commented:
This PR was merged into the repository. The changes were merged into the following branches:

- main: f5b50ec20d0855b75a89cbf0028f6f79bff2ab9c
